### PR TITLE
add domain js.cool

### PR DESCRIPTION
**Pull request by willin**

### CODE REVIEW

**Comment by ChristianHeimke** [comment]
The link is broken - I guess the correct url is https://github.com/willin/js.cool as far as I can see in willin's repositories..

**Comment by willin** [comment]
oh, i'm sorry

File: README.md [hunk 1]
```diff
@@ -1053,7 +1053,7 @@ Table of Contents
   * [freenom](https://freenom.com) — Free .tk,.ml,.ga,.cf and .gq domain.
   * [eu.org](https://nic.eu.org) — Free eu.org domain. Request is usually approved in 14 days.
   * [js.org](https://js.org) - Free js.org subdomains for GitHub Pages for the JavaScript community.
-  * [js.cool](https://github.com/willin/js-cool) - Free js.cool subdomains for GitHub Pages and Vercel.
+  * [js.cool](https://github.com/willin/js.cool) - Free js.cool subdomains for GitHub Pages and Vercel.
```
Comment (ripienaar) [inline README.md:1056]: It’s nice and all but I can’t really add it as this is just too restrictive. 


“ Your site must have a Chinese translation edition”

**Comment by willin** [comment]
Your site ~~must~~(will be nice to) have a Chinese translation edition

File: README.md (hunk 1 above)
Comment (ripienaar) [reply README.md:1056]: Are these actually CNAMEs as stated? It doesn’t seem that way to me. It seems like a system of redirects rather than CNAME?

Comment (willin) [reply README.md:1056]: - https://retrounhash.js.cool/
- https://leader.js.cool/
- https://xrkffgg.js.cool/
- and so on

Comment (ripienaar) [reply README.md:1056]: Yes I saw and none of these are CNAMEs they resolve to the same IP.

Comment (ripienaar) [reply README.md:1056]: Compare with for example js.org - that one makes actual CNAME dns entries. This one does not do  that.

Comment (ripienaar) [reply README.md:1056]: To be clear: I am not listing a redirect based service.

Comment (willin) [reply README.md:1056]: image


https://github.com/willin/js.cool/blob/master/main/active_cname.js

Comment (willin) [reply README.md:1056]: perhaps because proxied?

Comment (ripienaar) [reply README.md:1056]: Yes.

Comment (ripienaar) [reply README.md:1056]: Others like js.org are not proxies

Comment (willin) [reply README.md:1056]: same with js.org, it depends on users' willing: https://github.com/js-org/js.org/blob/master/cnames_active.js#L17-L22

Comment (ripienaar) [reply README.md:1056]: Suggest you show this in similar comments in readme or here and I would need an example that works

Comment (willin) [reply README.md:1056]: and it's almost the same implementation.

**Comment by ripienaar** [comment]
Thank you

**Pull request merged**